tests/db/jobs/test_jobcontroller.py

```diff
@@ -1112,6 +1112,22 @@ def test_running_runner(job_controller, daemon_manager, wait_daemon_started):
     assert job_controller.get_running_runner() == "NO_DOCUMENT"
 
 
+def test_runner_pings(job_controller, monkeypatch):
+    data = {"test": "XX"}
+    import jobflow_remote.jobs.jobcontroller
+
+    with monkeypatch.context() as m:
+        m.setattr(jobflow_remote.jobs.jobcontroller, "_MAX_ARCHIVED_RUNNER_PINGS", 3)
+        job_controller.ping_running_runner(data=data)
+        job_controller.ping_running_runner(data=data)
+        assert len(job_controller.get_runner_pings()) == 2
+        for _ in range(3):
+            job_controller.ping_running_runner(data=data)
+        assert len(job_controller.get_runner_pings()) == 3
+        job_controller.clean_runner_pings()
+        assert len(job_controller.get_runner_pings()) == 0
```

Comment:
Check that it is that it is the last three that are there ?

```suggestion
    import jobflow_remote.jobs.jobcontroller

    with monkeypatch.context() as m:
        m.setattr(jobflow_remote.jobs.jobcontroller, "_MAX_ARCHIVED_RUNNER_PINGS", 3)
        job_controller.ping_running_runner(data={"test": "1"})
        job_controller.ping_running_runner(data={"test": "2"})
        assert len(job_controller.get_runner_pings()) == 2
        for ii in range(3):
            job_controller.ping_running_runner(data={"test": str(ii+3)})
        pings = job_controller.get_runner_pings()
        assert len(pings) == 3
        test_numbers = [ping["test"] for ping in pings]
        assert set(test_numbers) == {"3", "4", "5"}
        job_controller.clean_runner_pings()
        assert len(job_controller.get_runner_pings()) == 0
```

Reply:
done, but I also checked the order